packages/renderer/src/lib/recommendation/ExtensionBanner.svelte

```diff
@@ -46,6 +66,9 @@ $effect(() => {
 
   <!-- feature extension actions -->
   <div class="flex flex-col">
+    <div class="flex flex-row justify-end">
+      <CloseButton on:click={onClose} />
```

Comment:
![image](https://github.com/user-attachments/assets/5f6e255f-27af-48d0-9650-fe5823528dd6)

The close button is not visible in light mode

Might be interesting to have a color property on the CloseButton, currently there is not way to avoid the `text-gray-800`.

https://github.com/podman-desktop/podman-desktop/blob/516922e2ef043c208d55e9921edda79f7798aa20/packages/ui/src/lib/button/CloseButton.svelte#L15

A nice way would be to use component scoped variable (here is an example https://svelte.dev/playground/bb42c17d585b4236824e1d5e48fe92b5?version=3.38.0)

we could add `--text-color`, `--text-hover` etc.?

Reply:
I've created a PR for setting color properties of Close button